due-processed/template-parts/icon-email.php

```diff
@@ -0,0 +1,3 @@
+<svg class="icon" width="18" height="12" viewBox="0 0 18 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+	<path d="M17.9927 0.326416L-0.00732422 0.355216L0.00707578 11.6736L18.0071 11.6448L17.9927 0.326416ZM16.6463 0.981616L8.99988 7.27442L1.33908 1.00322L16.6463 0.981616ZM0.655076 11.0256L0.640676 1.26962L8.99988 8.10962L17.3447 1.24082L17.3591 10.9968L0.655076 11.0256Z" fill="black"/>
```

Comment:
remove `fill` at end
```suggestion
	<path d="M17.9927 0.326416L-0.00732422 0.355216L0.00707578 11.6736L18.0071 11.6448L17.9927 0.326416ZM16.6463 0.981616L8.99988 7.27442L1.33908 1.00322L16.6463 0.981616ZM0.655076 11.0256L0.640676 1.26962L8.99988 8.10962L17.3447 1.24082L17.3591 10.9968L0.655076 11.0256Z"/>
```